modules/cluster-logging-forwarding-json-logs-to-the-default-elasticsearch.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-forwarding-json-logs-to-the-default-elasticsearch_{context}"]
+= Forwarding JSON logs to the default Elasticsearch log store
+
+For the default Elasticsearch log store that OpenShift Logging manages, you must create a different index for each format in advance if your JSON log entries _have different formats_. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+
+.Procedure
+
+. Add the following snippet to your `ClusterLogForwarder` CR YAML file.
++
+[source,yaml]
+----
+outputDefaults:
+- elasticsearch:
+    structuredTypeKey: <log record field>
+    structuredTypeName: <name>
+pipelines:
+- inputRefs:
+  - application
+  outputRefs: default
+  parse: json <4>
+----
+
+. Optional: Use `structuredTypeKey` to specify one of the log record fields, as described in the preceding topic, xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_cluster-logging-enabling-json-logging[Configuration of JSON log data for default Elasticsearch]. Otherwise, remove this line.
+
+. Optional: Use `structuredTypeName` to specify a `<name>`, as described in the preceding topic, xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_cluster-logging-enabling-json-logging[Configuration of JSON log data for default Elasticsearch]. Otherwise, remove this line.
+
+[IMPORTANT]
+====
+To forward JSON logs, you must set either `structuredTypeKey` or `structuredTypeName`, or both  `structuredTypeKey` and `structuredTypeName`.
+====
+
+. For `inputRefs`, specify which log types should be forwarded using that pipeline, such as `application,` `infrastructure`, or `audit`.
```

Comment:
You need plus signs (+) above and below your admonition so that this step is correctly numbered as step 4 (currently is set to step 1 in the output).

Reply:
Updated. Thanks.